<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_sup">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_title">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;Century Schoolbook&quot;;</string>
         </property>
         <property name="text">
          <string>ventana Title</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>914</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_minimizar">
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/source/minimizar.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_minmax">
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/source/restaurar.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cerrar">
         <property name="minimumSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>15</width>
           <height>15</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/source/cerrar.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_body">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0,177);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,7">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="sidebar">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 100);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="styleSheet">
              <string notr="true">
background-color: rgba(195, 195, 195,0);
border:none;</string>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>822</width>
                <height>702</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QFrame" name="frame_2">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="7,1">
                  <item>
                   <widget class="QFrame" name="frame_3">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(255, 255, 255, 90);</string>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::Box</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Sunken</enum>
                    </property>
                    <property name="lineWidth">
                     <number>1</number>
                    </property>
                    <property name="midLineWidth">
                     <number>1</number>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_3">
                       <property name="spacing">
                        <number>20</number>
                       </property>
                       <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                         <property name="leftMargin">
                          <number>9</number>
                         </property>
                         <item>
                          <widget class="QLabel" name="label_buscar_clientes">
                           <property name="font">
                            <font>
                             <family>Century Schoolbook</family>
                             <pointsize>15</pointsize>
                            </font>
                           </property>
                           <property name="styleSheet">
                            <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
                           </property>
                           <property name="text">
                            <string>Cedula:</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QLineEdit" name="lE_buscar_clientes">
                           <property name="font">
                            <font>
                             <family>Century Schoolbook</family>
                             <pointsize>10</pointsize>
                            </font>
                           </property>
                           <property name="styleSheet">
                            <string notr="true">border:none;
border-bottom:1px solid #fff;
background-color: rgba(255, 255, 255, 0);</string>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <widget class="QPushButton" name="btn_buscar_clientes">
                           <property name="minimumSize">
                            <size>
                             <width>24</width>
                             <height>24</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>24</width>
                             <height>24</height>
                            </size>
                           </property>
                           <property name="styleSheet">
                            <string notr="true">QPushButton{
  background-color: rgba(255, 255, 255, 0);
  border:none;
}
QPushButton:hover{
 border-bottom: 2px solid #ffffff;
 border-radius: 12;
}</string>
                           </property>
                           <property name="text">
                            <string/>
                           </property>
                           <property name="icon">
                            <iconset resource="../../static/recursos.qrc">
                             <normaloff>:/source/lupa.png</normaloff>:/source/lupa.png</iconset>
                           </property>
                           <property name="iconSize">
                            <size>
                             <width>15</width>
                             <height>15</height>
                            </size>
                           </property>
                          </widget>
                         </item>
                         <item>
                          <spacer name="horizontalSpacer_2">
                           <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                           </property>
                           <property name="sizeType">
                            <enum>QSizePolicy::Expanding</enum>
                           </property>
                           <property name="sizeHint" stdset="0">
                            <size>
                             <width>200</width>
                             <height>20</height>
                            </size>
                           </property>
                          </spacer>
                         </item>
                        </layout>
                       </item>
                       <item>
                        <widget class="QListWidget" name="tw_clientes">
                         <property name="styleSheet">
                          <string notr="true">QListWidget{
	background-color: rgba(255, 255, 255,0);
}

QListWidget::item:hover {
	background-color: #fff;
}
QListWidget::item:selected {
	background-color: rgba(0,255,0,80);
}</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_list_btn_clientes">
                    <property name="minimumSize">
                     <size>
                      <width>200</width>
                      <height>81</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgba(255, 255, 255, 90);</string>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1,0">
                     <property name="spacing">
                      <number>100</number>
                     </property>
                     <property name="leftMargin">
                      <number>10</number>
                     </property>
                     <property name="topMargin">
                      <number>50</number>
                     </property>
                     <property name="rightMargin">
                      <number>10</number>
                     </property>
                     <property name="bottomMargin">
                      <number>10</number>
                     </property>
                     <item>
                      <widget class="QPushButton" name="pushButton">
                       <property name="text">
                        <string>PushButton</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButton_2">
                       <property name="text">
                        <string>PushButton</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButton_3">
                       <property name="text">
                        <string>PushButton</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="verticalSpacer">
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>261</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QScrollArea" name="scrollArea_3">
             <property name="styleSheet">
              <string notr="true">background-color: rgba(255, 255, 255,0);</string>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>98</width>
                <height>28</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6"/>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <layout class="QVBoxLayout" name="verticalLayout_5"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../static/recursos.qrc"/>
 </resources>
 <connections/>
</ui>
